packages/manager/modules/pci/src/components/hds/hds.constant.js

```diff
@@ -0,0 +1,21 @@
+export const HDS_GET_MORE_INFO = {
```

Comment:
For info this is a temporary list.
We wait for the final list.

Reply:
Don't forget `US` sub in the definitive list :)

Reply:
This feat is only for EU and CA, do we must add it ?

Reply:
Then no :) I didn't notice any feature flipping mechanism, how is checking the region handled

Reply:
I add it. thanks.